spec/models/manageiq/providers/kubernetes/container_manager/scanning/job_spec.rb

```diff
@@ -112,7 +112,7 @@ def fetch_oscap_arf
         @job.signal(:data, '<summary><syncmetadata></syncmetadata></summary>')
       end
 
-      @job = @ems.raw_scan_job_create(@image)
+      @job = @ems.raw_scan_job_create(@image.class.name, @image,id)
```

Comment:
I think you typo-ed  a comma for a point ( `@image.id` ) and this might also be the reason the tests are failing